backend/orchestrator/computetask_pb2.pyi

```diff
@@ -346,9 +345,6 @@ class NewComputeTask(google.protobuf.message.Message):
     compute_plan_key: builtins.str
     worker: builtins.str
     @property
-    def parent_task_keys(self) -> google.protobuf.internal.containers.RepeatedScalarFieldContainer[builtins.str]:
```

Comment:
Could we seperate both feature removal of compute plan status and parent_task_key ?

Reply:
Oops! Nice catch! I updated the PR to only alter the compute plan protobuf client.